dashboards-observability/common/constants/application_analytics.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+export const TAB_OVERVIEW_ID_TXT_PFX = 'app-analytics-overview-';
```

Comment:
is PFX = prefix?

Reply:
I assume yes, I brought this over from explorer, but taking another look, I may not need this cause it was only used to create unique IDs